KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -59,9 +60,18 @@ projectFile=''
                     testArgs = " -parallel none";
                 }
 
-                // Dotnet("test" + testArgs, projectFolder);
+                if (!IsLinux ||
+                    string.IsNullOrEmpty(KOREBUILD_TEST_SKIPMONO) ||
+                    string.Equals(KOREBUILD_TEST_SKIPMONO, "0") ||
+                    string.Equals(KOREBUILD_TEST_SKIPMONO, "false"))
+                {
+                    // Dotnet("test" + testArgs, projectFolder);
+                }
             }
-            else if (!IsLinux || !string.IsNullOrEmpty(KOREBUILD_TEST_DNXCORE))
+            else if (!IsLinux ||
+                     (!string.IsNullOrEmpty(KOREBUILD_TEST_DNXCORE) &&
+                      !string.Equals(KOREBUILD_TEST_DNXCORE, "0") &&
+                      !string.Equals(KOREBUILD_TEST_DNXCORE, "false")))
```

Comment:
Also, I don't think it's so odd. Handling `0` and `false` as false and all else as true is somewhat like what some programming languages do.